src/TraceBundle.php

```diff
@@ -2,18 +2,18 @@
 
 declare(strict_types=1);
 
-namespace DR\SymfonyRequestId;
+namespace DR\SymfonyTraceBundle;
 
-use DR\SymfonyRequestId\DependencyInjection\Compiler\HttpClientTraceIdPass;
-use DR\SymfonyRequestId\DependencyInjection\SymfonyRequestIdExtension;
+use DR\SymfonyTraceBundle\DependencyInjection\Compiler\HttpClientTraceIdPass;
+use DR\SymfonyTraceBundle\DependencyInjection\SymfonyTraceExtension;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
 /**
  * @codeCoverageIgnore - This is a bundle class, tested by the functional test
  */
-final class RequestIdBundle extends Bundle
+final class TraceBundle extends Bundle
```

Comment:
Why don't we follow the symfony naming standard here? Symfony Flex can automatically add a bundle to the bundles.php configuration when installing as long as the name follows the standard. https://symfony.com/doc/current/bundles/best_practices.html#bundle-name

Reply:
"Can automatically" or "will automatically via contrib repository" ?

Im not familiar with the features of the flex recipe, but looking at the contrib repository, I mostly see deviations from the naming standard pattern. 

https://github.com/symfony/recipes-contrib/blob/main/drinksco/console-ui-bundle/0.2/manifest.json
https://github.com/symfony/recipes-contrib/blob/main/php-http/httplug-bundle/1.6/manifest.json
https://github.com/symfony/recipes-contrib/blob/main/jrmgx/interactive-bundle/1.0/manifest.json

Imo, the naming strategy is kinda awful, and I think an addition to the contrib repository will work much better.

Reply:
If the bundle name follows the standard symfony flex will automatically enable the bundle without a contrib recipe

Reply:
Okay, that's a benefit. If I understand the documentation correctly it should be based on the namespace. 

So:
package name: `digitalrevolution/trace-bundle`
namespace: `DR\TraceBundle\`
the bundle class name then becomes: `DR\TraceBundle\DRTraceBundle`.

Correct?

Reply:
SymfonyTraceBundle.php seems to work on a local symfony-skeleton project.
I prefer that over DRSymfonyTraceBundle / DigitalRevolutionSymfonyTraceBundle

Reply:
> Okay, that's a benefit. If I understand the documentation correctly it should be based on the namespace.
> 
> So: package name: `digitalrevolution/symfony-trace-bundle` namespace: `DR\SymfonyTraceBundle\` the bundle class name then becomes: `DR\SymfonyTraceBundle\DRSymfonyTraceBundle`.
> 
> Correct?

Yes that looks correct

Reply:
Tested it myself aswell. Looks good